Icon-Legend.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+
+    <link rel="stylesheet" href="css/main.css">
+</head>
+<body>
+<!-- 
+    website Refrence for icons 
+    https://www.wta.org/go-hiking/hikes/surprise-lake-1 -->
+
+    <div class="legend">
+
+
+    <table >
+        <col  style="background-color:rgb(0, 255, 213)">
+        <col style="background-color:rgb(172, 158, 255)">
+      </colgroup>
+        <tr>
+            <th> Icon  </th>
+            <th> Description </th>
+
+        </tr>
+
+        <tr>
+            <th > <img src="icon-images/Parking.png" alt="limited-parking">  </th>
+            <th>  Parking Avaliable </th>
```

Comment:
...and should this be a <td> tag because it is a data cell under the "Description" header for the table? (same question applies for the rest of the icons/descriptions in the table)
```suggestion
            <td>  Parking Avaliable </td>
```

Reply:
I thought about that, but I wanted the table to just go sideways so i left it as th and th, if I did th and td it would go by Columns and leave a lot of unused space..... Unless im missing something with table drawing.